tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -98,30 +114,48 @@ def montecarlo_radial1d(
     runner.last_line_interaction_out_id = last_line_interaction_out_id
 
     if montecarlo_configuration.VPACKET_LOGGING and number_of_vpackets > 0:
+        # Log message To be determined correctly
+        logger.debug(
+            "Virtual Packet Logging Turned ON\n\tDetails for Virtual Packet Logging"
+        )
         runner.virt_packet_nus = np.concatenate(
-            np.array(virt_packet_nus)
+            np.array(
+                virt_packet_nus,
+            ),
         ).ravel()
+        # logger.debug(f"Virtual Packet nu's = {runner.virt_packet_nus}")
```

Comment:
@andrewfullard please confirm about the status of the log messages. Are they useful?